dotnet/Sample/Sample1_AnalyzeText.cs

```diff
@@ -0,0 +1,44 @@
+using Azure.AI.ContentSafety;
+
+namespace Azure.AI.ContentSafety.Dotnet.Sample
+{
+    class ContentSafetySampleAnalyzeText
+    {
+        public void AnalyzeText()
+        {
+            #region Snippet:Azure_AI_ContentSafety_CreateClient
```

Comment:
remove all the #region and #endregion lines, and replace with readable comments.
Like // create client